src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -102,9 +102,9 @@ public async Task<HttpResponse> SendRequestAsync(
                 timeoutException = exception;
             }
 
-            while (_retryPolicy.pauseForRetry(response, timeoutException, retryCount))
+            while (await _retryPolicy.PauseForRetryAsync(response, timeoutException, retryCount, logger).ConfigureAwait(false))
```

Comment:
can we pass the caller’s CancellationToken into PauseForRetryAsync?

Reply:
Why would we do that if it's not being done already?

Reply:
It's not used in the retry policy